src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -736,7 +736,7 @@ We want everyone to monitor their systems, and we're contributing our technology
 
 [New Relic Open Source](https://opensource.newrelic.com/explore-projects/) projects are assigned to one of five different categories. These categories determine the support options available for a project as listed below:
 
-* **Community plus projects**: Actively maintained by New Relic. Support requests can be made through Github, Community, and Ticketed Support channels, depending on the service level associated with the New Relic account.
+* **Community plus projects**: Actively maintained by New Relic. Support requests can be made through Github, Community, and Case Support channels, depending on the service level associated with the New Relic account.
```

Comment:
Should 'case support channels' be all lowercase? It's more abstract/general than the more specific channels listed in the table.

Reply:
We tend to let support handle styling of these docs